ghost/core/core/server/data/migrations/versions/5.61/2023-08-29-09-32-32-add-recommendations-enabled-setting.js

```diff
@@ -0,0 +1,8 @@
+const {addSetting} = require('../../utils');
+
+module.exports = addSetting({
+    key: 'recommendations_enabled',
+    value: 'true',
```

Comment:
```suggestion
    value: 'false',
```

Reply:
Updated, following the Slack discussion [here](https://ghost.slack.com/archives/C04TMVA1D7A/p1693310637853019)